SCons/Environment.py

```diff
@@ -1966,12 +1977,11 @@ def PrependUnique(self, delete_existing: bool=False, **kw) -> None:
                     val = [x for x in val if x not in dk]
                 self._dict[key] = val + dk
             else:
+                # val is not a list, so presumably a scalar (likely str).
```

Comment:
Do the list comprehensions at 1975 and 1977 also need this fix?
(and 1991?)

Reply:
didn't think so because `val` is a list in that context - the membership test should be the right one there.  However, I'll have a think at it (tomorrow) and see if I could be wrong....

Reply:
Those are correct as they are.